pkg/util/filesystem/util_windows.go

```diff
@@ -85,3 +87,10 @@ func IsUnixDomainSocket(filePath string) (bool, error) {
 	}
 	return true, nil
 }
+
+// IsAbs returns wether the given path is absolute or not.
```

Comment:
```suggestion
// IsAbs returns whether the given path is absolute or not.
```

Reply:
Fixed in cefe0b2ed04806186ecc671474bf2dbece4857ae